api_tests/e2e/rfc003/btc_eth/happy.ts

```diff
@@ -2,283 +2,153 @@ import * as bitcoin from "../../../lib/bitcoin";
-                    quantity: alpha_asset_quantity.toString(),
+                5000
+            ),
+        }),
+        new ActionTrigger({
+            actor: bob,
+            action: Action.Redeem,
+            timeout: 10000,
+            parameters: "address=" + bobFinalAddress + "&fee_per_byte=20",
+            afterTest: new AfterTest(
+                "[bob] Should have received the alpha asset after the redeem",
+                async function(swapLocations: { [key: string]: string }) {
+                    let body = (await bob.pollComitNodeUntil(
+                        swapLocations["bob"],
+                        body => body.state.alpha_ledger.status === "Redeemed"
+                    )) as SwapResponse;
+                    let redeemTxId = body.state.alpha_ledger.redeem_tx;
+
+                    let satoshiReceived = await bitcoin.getFirstUtxoValueTransferredTo(
+                        redeemTxId,
+                        bobFinalAddress
+                    );
+                    const satoshiExpected = alphaAssetQuantity - alphaMaxFee;
+
+                    satoshiReceived.should.be.at.least(satoshiExpected);
                 },
-                beta_asset: {
-                    name: "Ether",
-                    quantity: beta_asset_quantity.toString(),
-                },
-                beta_ledger_redeem_identity: alice_final_address,
-                alpha_expiry: alpha_expiry,
-                beta_expiry: beta_expiry,
-                peer: bob_comit_node_address,
-            })
-            .then(res => {
-                res.should.have.status(201);
-                swap_location = res.header.location;
-                swap_location.should.be.a("string");
-                alice_swap_href = swap_location;
-            });
-    });
-
-    it("[Alice] Should be in IN_PROGRESS and SENT after sending the swap request to Bob", async function() {
-        this.timeout(10000);
-        await alice.poll_comit_node_until(
-            alice_swap_href,
-            body =>
-                body.status === "IN_PROGRESS" &&
-                body.state.communication.status === "SENT"
-        );
-    });
-
-    let bob_swap_href: string;
+                10000
+            ),
+        }),
+    ];
 
-    it("[Bob] Shows the Swap as IN_PROGRESS in /swaps", async () => {
-        let body: any = (await bob.poll_comit_node_until(
-            "/swaps",
-            body => body._embedded.swaps.length > 0
-        )) as SwapsResponse;
-        let swap_embedded = body._embedded.swaps[0];
-        swap_embedded.protocol.should.equal("rfc003");
-        swap_embedded.status.should.equal("IN_PROGRESS");
-        let swap_link = swap_embedded._links;
-        swap_link.should.be.a("object");
-        bob_swap_href = swap_link.self.href;
-        bob_swap_href.should.be.a("string");
+    describe("RFC003: Bitcoin for Ether", async () => {
+        execute(alice, bob, actions, initialUrl, listUrl, swapRequest);
```

Comment:
:wrench: This method really just creates the test steps right? Can we name it according to that? :)